### PR TITLE
Update QuPath ROIsplitter

### PR DESCRIPTION
**Pull request by npinter**
Update to v0.2.0 (--fill and --all arguments added). See https://github.com/npinter/ROIsplitter.

### CODE REVIEW

File: tools/qupath_roi_splitter/qupath_roi_splitter.xml [hunk 1]
```diff
@@ -1,20 +1,35 @@
...
         #for $input in $input_collection
-            python3 '$__tool_directory__/qupath_roi_splitter.py' --qupath_roi '$input' &&
+            python3 '$__tool_directory__/qupath_roi_splitter.py'
+                    --qupath_roi '$input'
+                    #if $optional.fill
+                        --fill
+                    #end if
+                    #if $optional.all
+                        --all
+                    #end if
```
Comment (bgruening) [inline tools/qupath_roi_splitter/qupath_roi_splitter.xml:22]:
```suggestion
                    $optional.fill
                    $optional.all
```

**Comment by npinter** [comment]
I do not see what's wrong here?!

**Comment by bgruening** [comment]
> .. WARNING: No citations found, consider adding citations to your tool.

**Comment by npinter** [comment]
> > .. WARNING: No citations found, consider adding citations to your tool.

what if there is no citation? it sounded optional to me ("consider")

**Comment by bgruening** [comment]
You can include a bibtex citing yourself.

**Review by bgruening — APPROVED** [review]

**Pull request merged**